pkg/cri/sbserver/sandbox_run_test.go

```diff
@@ -135,7 +135,7 @@ func TestSelectPodIP(t *testing.T) {
 			expectedAdditionalIPs: []string{"2001:db8:85a3::8a2e:370:7334"},
 		},
 		"ipv6 should be picked even if ipv4 comes first": {
-			ips:                   []string{"2001:db8:85a3::8a2e:370:7334", "192.168.17.43"},
+			ips:                   []string{"192.168.17.43", "2001:db8:85a3::8a2e:370:7334"},
```

Comment:
Minor conflict in the tests in adjacent lines because `main` rewrote these to table-tests using a slice instead of a map.